Server/config/passport.config.js

```diff
@@ -1,13 +1,49 @@
+const passport = require('passport');
+var LocalStrategy = require('passport-local').Strategy;
+const passportJWT = require('passport-jwt');
+const JWTStrategy = passportJWT.Strategy;
```

Comment:
we can declare it also similar to local strategy avoiding extra variable
`const JWTStrategy = require('passport-jwt').Strategy`

Reply:
Updated